service/iot-service-samples/device-method-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/service/sdk/DirectMethodSample.java

```diff
@@ -35,6 +35,13 @@
     private static final String methodName = "[Function Name]";
     private static final int responseTimeout = 200;
     private static final int connectTimeout = 5;
+
+    // The payload can be:
+    // Null: the "payload" field will not be included;
+    // Primitive type/String/Array/List/Map/custom type: will be serialized as value of the "payload" field using GSON.
+    //
+    // For a full list of end-to-end tests with different payload types, please refer to
+    // https://github.com/Azure/azure-iot-sdk-java/blob/main/iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/methods/DirectMethodsTests.java
```

Comment:
The link is available now, but the referring end-to-end tests will be available only after changes in this PR get merged.